courses/fundamentals_of_ada/060_record_types.rst

```diff
@@ -240,23 +251,31 @@ Aggregates
 * Literal values for composite types
 
    - As for arrays
+   - Default value / selector: :ada:`<>`, :ada:`others`
+
+* Can use both **named** and **positional**
+
+    - Unambiguous
 
 * Syntax (simplified):
 
    .. code:: Ada
 
-      record_aggregate ::= (record_component_association_list)
-      
-      record_component_association_list ::= 
-         record_component_association
-         { , record_component_association}
-      
-      record_component_association ::= 
-         [component_choice_list =>] expression
-      
-      component_choice_list ::= 
-         component_selector_name { | component_selector_name}
-         | others
+      record_expr ::= expression | <>
```

Comment:
Not "record_expr" - something more like "component_initialization"

Reply:
I made it to be `component_init` because of the large horizontal size otherwise